speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grboard.cpp

```diff
@@ -1615,13 +1658,27 @@ void grInitBoardCar(tCarElt *car)
   if (bMasterModel)
     lg += snprintf(grFilePath + lg, nMaxTexPathSize - lg, "cars/models/%s;", car->_masterModel);
 
-  lg += snprintf(grFilePath + lg, nMaxTexPathSize - lg, "data/textures");
+  lg += snprintf(grFilePath + lg, nMaxTexPathSize - lg, "data/textures;");
+
+  // Add the data/intervention folder to the searchable filepaths for filenames.
+  lg += snprintf(grFilePath + lg, nMaxTexPathSize - lg, "data/intervention");
+
+  // Load intervention texture from XML file, for now: UNSAFE with respect to path size (max is 256) 
+  char buf[256];
+  snprintf(buf, sizeof(buf), "%sdata/intervention/intervention.xml", GfDataDir());
+  void* xmlHandleIntervention = GfParmReadFile(buf, GFPARM_RMODE_STD);
+
+  // Search the xml file for the file corresponding to the attribute "steer intervention"
+  // Note that grSsgLoadTexState uses a buffer of 256 internally, so the file should not be bigger than 256x256
+  const char* attr = GfParmGetStr(xmlHandleIntervention, SECT_GROBJECTS, "steer intervention", NULL);
+  interventionTexture = (ssgSimpleState*)grSsgLoadTexState(attr);
+
```

Comment:
// SIMULATED DRIVING ASSISTANCE CHANGE:
here maybe not on every line, but make clear what has been changed